src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -160,6 +160,31 @@ def mgmt_if_entry_table_state_db(if_name):
 
     return b'MGMT_PORT_TABLE|' + if_name
 
+def get_sai_id_key(namespace, sai_id):
+    """
+    inputs:
+    namespace - string
+    sai id - bytes
+    Return type:
+    bytes
+    Return value: namespace:sai id or sai id
+    """
+    if namespace != '':
+        sai_id_key =  namespace + ':' + sai_id.decode()
```

Comment:
>namespace + ':' + sai_id.decode() [](start = 22, length = 33)

`return namespace.encode() + b':' + sai_id`
This will be easier and shorter.

Reply:
updated as per comment.